src/_posts/2020-03-24-project-maturity-model.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+asset-type: post
+name: project-maturity-model
+title: Project Maturity Model
+date: 2020-02-24 10:00:00 +00:00
+author: Christopher Eyre
+image:
+    src: /assets/custom/img/blog/railway.png
+    attribution:
+        text: Public domain photo from Pixabay
+        href: https://pixabay.com/photos/rails-soft-gleise-railway-3309912/
+tags:
+    - project improvements
```

Comment:
Please use a tag from the existing list. Use a minimum of 3 tags

Reply:
done.